src/compas/geometry/brep/face.py

```diff
@@ -0,0 +1,172 @@
+from compas.data import Data
+
+
+class BrepFace(Data):
+    """
+    An interface for a Brep Face
+    """
```

Comment:
currently, properties will not show up in the documentation if they are documented in their own docstrings. to include them, please list them here under `Attributes`

Reply:
this means you can omit the docstring in the property definition...